app/controllers/news_controller.rb

```diff
@@ -31,9 +29,10 @@ def show
         .order_by_published
         .limit(25)
     )
-    @news_item_data = serialize_news_items(
-      NewsItem.published.find(params[:id])
-    )
+    @news_item = NewsItem.published.find(params[:id])
```

Comment:
probably doesn't need to be an instance variable if not used outside the method

Reply:
Ah nice thanks for the catch!